views/_includes/input-dates.pug

```diff
@@ -0,0 +1,24 @@
+mixin dateInputs(idPrefix, legendText, isoHintText, values)
+  div.date-container(class={['has-error']: errors && errors[`${idPrefix}Day`]})
+    fieldset.fieldset(role="group", aria-describedby=`${idPrefix}Hint`,)
+      legend
+        h2 #{__(legendText)}
+
+      span#dobHint.cra-form-message #{__('For Example:')} #{isoDateHintText(isoHintText)}
+
+      if errors
+        ul
+          each err in errors
+            li
+              +validationMessage(err.msg, err.param)
+      div.w-1-4.d--ib
+        label(for=`${idPrefix}Day`) #{__('Day')}
```

Comment:
"Prefix" is a nice idea, but I abandoned it previously because our "matching" code in the validation only works with the dobStuff: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/schemas/login.schema.js#L64-L65

Potential solutions: 
- Update the validations so that we can pass through prefixes somehow
- Keep everything as some generic looking `day`, `month`, `year` type thing
- Keep everything as `dobDay` all the time.

Reply:
Looks like removing the validation for leap years solves this: 👍